### PR TITLE
[MCA-946] Setup GitHub actions

### PR DESCRIPTION
**Pull request by nikolauska**
Added github actions for test and linter. Included updates to CODEOWNERS as weill

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1 +1 @@
-@eliasvaltteri @henrinie @michalksiezuk-nc @jererobles @vastamaki @nikolauska @biphobe @ViktorHegyvari @aashirjaved @Nissinen @PerttuSavolainen @lintula @iriekun
+* @eliasvaltteri @henrinie @michalksiezuk-nc @jererobles @vastamaki @nikolauska @biphobe @PerttuSavolainen @lintula @iriekun
```
Comment (Ben3eeE) [inline .github/CODEOWNERS]: Bohdan, Johanna and me are not in CODEOWNERS

Comment (nikolauska) [reply .github/CODEOWNERS]: Thanks for pointing out, could you provide usernames so I could add the missing people?

Comment (Ben3eeE) [reply .github/CODEOWNERS]: @Ben3eeE @BoRuDar 

It seems like Johanna is not in the nordcloud org yet?

Comment (nikolauska) [reply .github/CODEOWNERS]: @terotil Could add Johanna to Nordcloud org?

Comment (Ben3eeE) [reply .github/CODEOWNERS]: The username is @katjajohanna

Comment (nikolauska) [reply .github/CODEOWNERS]: I've actually removed Johanna and Tero as we should only have the developers included in the list. You and Bogdan have been added to list with couple other changes included

File: .github/workflows/on_pr_master.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      matrix:
+        node-version: [12.x, 14.x]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: npm ci
+    - run: npm run lint
```
Comment (vastamaki) [inline .github/workflows/on_pr_master.yml:22]: Are these missing one indentation?

File: .github/workflows/on_pr_master.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+      matrix:
+        node-version: [12.x, 14.x]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: npm ci
+    - run: npm run test
```
Comment (vastamaki) [inline .github/workflows/on_pr_master.yml:36]: Same as above :point_up:

**Review by vastamaki — COMMENTED** [review]
If indentations are okay then LGTM

File: .github/workflows/on_pr_master.yml (hunk 3 above)
Comment (nikolauska) [reply .github/workflows/on_pr_master.yml:36]: It might look like it, but it is not :)

File: .github/workflows/on_pr_master.yml (hunk 2 above)
Comment (nikolauska) [reply .github/workflows/on_pr_master.yml:22]: It might look like it, but it is not :)

**Review by vastamaki — APPROVED** [review]

**Pull request merged**